webots_ros2_driver/webots_ros2_driver/utils.py

```diff
@@ -135,19 +135,26 @@ def get_host_ip():
         sys.exit('Unable to get host IP address. \'ip route\' could not be executed.')
 
 
+def get_wsl_ip_address():
+    # wsl2 net mode
+    # NAT: if get_router_ip() == get_wsl_host_ip()
+    # Mirrored: if get_router_ip() != get_wsl_host_ip()
+    return get_wsl_host_ip() if get_router_ip() == get_wsl_host_ip() else '127.0.0.1'
```

Comment:
We should cache the result of calling this method as it is not lightweight:
```suggestion
    wsl_host_ip = get_wsl_host_ip()
    return wsl_host_ip if get_router_ip() == wsl_host_ip else '127.0.0.1'
```

Reply:
you're right